Cython/Compiler/ModuleNode.py

```diff
@@ -3555,14 +3555,19 @@ def generate_main_method(self, env, code):
             wmain = "wmain"
         else:
             wmain = Options.embed
+        p_embed_modules = (env.module_name,) + tuple(Options.embed_modules)
+        c_embed_modules = (env.module_name,) + tuple(mname.replace('.', '_') for mname in Options.embed_modules)
```

Comment:
This can be simplified using list unpacking. And I would prefer a list here rather than a tuple because we are dealing with a homogeneous sequence, not a struct-like collection of different values.
```suggestion
        p_embed_modules = [env.module_name, *Options.embed_modules]
        c_embed_modules = [env.module_name, *(mname.replace('.', '_') for mname in Options.embed_modules)]
```
However, this isn't quite how it works. I think we need the original package information in order to generate correct C code.